content/applications/finance/accounting/getting_started/process_overview/supplier_bill.rst

```diff
@@ -1,144 +1,108 @@
-
-.. image:: supplier_bill/vendor_bill05.png
-   :align: center
-
-To register a new vendor bill, start by selecting a vendor and inputting
-their invoice as the **Vendor Reference**, then add and confirm the product
-lines, making sure to have the right product quantities, taxes and
-prices.
-
-.. image:: supplier_bill/vendor_bill01.png
-   :align: center
+.. seealso::
+   - Tutorial `Registering a vendor bill <https://www.odoo.com/slides/slide/registering-a-vendor-bill-1683?fullscreen=1>`_
+   - :ref:`Manage-vendor-bills`
 
-Save the invoice to update the pre tax and tax amounts at the bottom of
-the screen. You will most likely need to configure the prices of your
-products without taxes as Odoo will compute the tax for you.
+Bills creation
+==============
 
-.. note::
-    On the bottom left corner, Odoo shows a summary table of all taxes on the vendor bill.
-    In several countries, different methods are accepted to round the totals (round per line,
-    or round globally). The default rounding method in Odoo is to round the final prices
-    per line (as you may have different taxes per product. E.g. Alcohol and cigarettes).
-    However if your vendor has a different tax amount on their bill, you can change the
-    amount in the bottom left table to adjust and match.
+From the Accounting app, vendor bills can either be created manually or automatically.
 
-Validate The Vendor Bill
-------------------------
+Manually
+--------
 
-Once the vendor bill is validated, a journal entry will be generated
-based on the configuration on the invoice. This journal entry may differ
-depending on the the accounting package you choose to use.
+It is possible to manually create a vendor bill by going to :menuselection:`Vendors --> Bills` and
+then clicking :guilabel:`Create`.
 
-For most European countries, the journal entry will use the following
-accounts:
+Automatically
+-------------
 
--  **Accounts Payable:** defined on the vendor form
+Vendor bills can be automatically created by **sending an email** to an :ref:`email alias
+<invoice-digitization/email-alias>` associated with the purchase journal, or by **uploading a .pdf**
+in :menuselection:`Accounting app --> Vendors --> Bills` and then clicking :guilabel:`Upload`.
 
--  **Taxes:** defined on the products and per line
+Bills completion
+================
 
--  **Expenses:** defined on the line item product used
+Whether you create the bill manually or if it is generated automatically, make sure the following
+fields are completed as appropriate:
 
-For Anglo-Saxon (US) accounting, the journal entry will use the
-following accounts:
-
--  **Accounts Payable:** defined on the vendor form
+.. image:: supplier_bill/bill-completion.png
+   :align: center
+   :alt: filling the vendor bill
 
--  **Taxes:** defined on the products and per line
+First, select the :guilabel:`Vendor`. Odoo automatically fills some fields based on the vendor's
```

Comment:
```suggestion
First, select the :guilabel:`Vendor`. Odoo automatically fills some fields based on the vendor's
```
If there's a "First", in the text, there has to be a "Second" at some point later. If not, you can simply say "Select the vendor [...]" :)